pages/contribution.md

```diff
@@ -51,8 +51,8 @@ If this is your first time contributing to the magazine, you should also add you
 
 ```diff
 [authors]
-rust-magazine = { name = "The Editor Team", editor = true, bio = "The magazine editor team" }
+rust-magazine = { name = "The Editorial Team", editor = true, bio = "The magazine editorial team" }
```

Comment:
Good catch, I think we also should update the root `zine.toml` file.

Reply:
Yes, good call.